Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: test integration
 We always want to run the tests even if our platform is not 
 in the database of validated platforms. And the NFS based
 test has no chance of success in a bare Debian machine.
Forwarded: not-needed
Last-Update: 2012-04-18
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -10,7 +10,7 @@
 
 HDR=	LsofTest.h
 
-CKTSTDB= CkTestDB
+CKTSTDB= true
 CONFCFL= ./config.cflags
 CONFIG=	./config.cc ${CONFCFL} ./config.xobj
 LTOBJ=	LTlib.o
@@ -19,13 +19,14 @@
 
 BASTST=	LTbasic
 STDTST=	LTnlink LTsock LTszoff LTunix
-OPTTST=	LTbigf LTdnlc LTlock LTnfs 
+OPTTST=	LTbigf LTdnlc LTlock
 
-all:	${CKTSTDB} ${BASTST} ${STDTST} FRC
-	@./${CKTSTDB}; xv=$$?; \
+all:	${BASTST} ${STDTST} FRC
+	@${CKTSTDB}; xv=$$?; \
 	if [ $$xv -ne 0 ]; then \
 	  exit 1 ;\
 	fi
+	@pwd
 	@rm -f config.LT*
 	-@err=0; \
 	echo ""; \
@@ -59,8 +60,8 @@
 
 auto:	ckDB silent FRC
 
-ckDB:	${CKTSTDB} FRC
-	@echo "" | ./${CKTSTDB}; xv=$$?; \
+ckDB:	FRC
+	@echo "" | ${CKTSTDB}; xv=$$?; \
 	if [ $$xv -ne 0 ]; then \
 	  exit 1 ;\
 	fi
@@ -110,7 +111,7 @@
 	`cat config.cc` ${CFLAGS} `cat config.cflags` LTunix.c \
 	${LIBOBJ} `cat config.xobj` -o LTunix `cat config.ldflags`
 
-opt:	${CKTSTDB} ${OPTTST} FRC
+opt:	${OPTTST} FRC
 	@rm -f config.LT*
 	-@err=0; \
 	echo ""; \
@@ -157,3 +158,4 @@
 std: all
 
 test: all
+
